instruqt-tracks/terraform-build-your-own-provider/track.yml

```diff
@@ -1072,11 +1074,13 @@ challenges:
     cd examples
     ```
 
-    Update your `HASHICUPS_USERNAME` and `HASHICUPS_PASSWORD` env vars so they use the wrong credentials.
+    Update the provider section in your `main.tf` so they use the wrong credentials
 
     ```
-    export HASHICUPS_USERNAME=education
-    export HASHICUPS_PASSWORD=test1234
+    provider "hashicups" {
+      username = "education"
+      password = "badpassword"
+    }
```

Comment:
Did we change the way the provider works? Or is this just a preference thing?

Reply:
The instructions didn't work as expected because the creds are hard coded into the main.tf.. this was the fastest fix - over finding the main.tf to update. But can track it down..